tiledb/tests/test_fragments.py

```diff
@@ -705,9 +705,9 @@ def write_fragments(target_path, dshape, num_writes):
             assert frags.timestamp_range == ts
 
         if use_timestamps:
-            tiledb.Array.delete_fragments(path, 3, 6)
+            tiledb.delete_fragments(path, 3, 6)
```

Comment:
This is a breaking change. I am not familiar with Python API design, but I prefer the method to be on `Array`.